content/tutorials/single-page-tutorials/proof-inclusion-of-a-block.md

```diff
@@ -0,0 +1,298 @@
+
+### Architecture
+
+In this use-case pattern, the party writing to the Tangle (**Prover**) is the same party reading from it, which allows reading a block by its block Id together with a proof. All a **Verifier** needs to verify this proof, is the chain of milestones back to the milestone that referenced this exact block. Every milestone directly references the previous milestone, which leads to a unique and verifiable chain of milestones in the Tangle. This means even if the **Verifier's** notarization plugin didn't record the milestone history already, it could still download the milestone chain from another source and verify its correctness, before using it for further notarization verifications.
+
+Since the pattern assumes trusted access to the network, it is strongly recommended that both, the **Prover** and the **Verifier** operate their own network node and run the [INX Notarization/PoI Plugin](https://github.com/iotaledger/inx-poi). It is also noteworthy that this pattern does not rely on a Permanode storing the full Tangle history, which drastically reduces the storage requirements on the node operator side.
+
+![PoI-Architecture](proof-inclusion-of-a-block-architecture.png)
+
+### Prerequisites
+
+- Network
+
+  - Running [Hornet Private Tangle](https://github.com/iotaledger/hornet/tree/develop/private_tangle) (Branch **develop** for the latest [Stardust](https://blog.shimmer.network/stardust-upgrade-in-a-nutshell/) features)
+  - The node(s) will start with the necessary [INX-POI](https://github.com/iotaledger/inx-poi) plugin by default
+
+- Development Environment and Libraries
+  - Installed [Node.js](https://nodejs.org/en/)
+
+### Set Up
+
+#### Prepare development environment
+
+Create a new folder for the tutorial in a location of your choice.
+
+```bash
+mkdir notarization-tutorial
+cd notarization-tutorial
+```
+
+Now we need to run the Node.js initializer to configure the project.
+
+```bash
+npm init --yes
+```
+
+Add the following dependencies to the `package.json` file.
+
+```yaml
+{
+  'name': 'proof-of-inclusion',
+  'version': '1.0.0',
+  'description': 'Proof of Inclusion to the Tangle',
+  'main': 'create-notarization.js',
+  'dependencies':
+    {
+      '@iota/iota.js': 'next'
+    },
+  'scripts': { 'test': 'echo "Error: no test specified" && exit 1' },
+  'author': '',
+  'license': 'Apache-2.0',
+}
+```
+
+Install dependencies.
+
+```bash
+npm install
+```
```

Comment:
````suggestion
## Set Up

### Prepare Your Development Environment

1. Create a new folder for the tutorial in a location of your choice, for example:

```bash
mkdir notarization-tutorial
cd notarization-tutorial
```

2.  You should run the Node.js initializer to configure the project.

```bash
npm init --yes
```

3. Replace the content of the `package.json` with the following:

```json
{
  'name': 'proof-of-inclusion',
  'version': '1.0.0',
  'description': 'Proof of Inclusion to the Tangle',
  'main': 'create-notarization.js',
  'dependencies':
    {
      '@iota/iota.js': 'next'
    },
  'scripts': { 'test': 'echo "Error: no test specified" && exit 1' },
  'author': '',
  'license': 'Apache-2.0',
}
```

4. Install dependencies by running the following command:

```bash
npm install
```
````